cdap-examples/FileSetExample/src/main/java/co/cask/cdap/examples/fileset/FileSetService.java

```diff
@@ -106,6 +109,7 @@ public void read(HttpServiceRequest request, HttpServiceResponder responder,
      */
     @PUT
     @Path("{fileset}")
+    @TransactionPolicy(TransactionControl.EXPLICIT)
```

Comment:
This just seemed like a bad example to me, because it doesn't require this annotation here.